pkg/master/master.go

```diff
@@ -197,29 +197,16 @@ func setDefaults(c *Config) {
 		// Find and use the first non-loopback address.
 		// TODO: potentially it'd be useful to skip the docker interface if it
 		// somehow is first in the list.
-		addrs, err := net.InterfaceAddrs()
+		hostIP, found, err := util.ChooseHostInterface([]string{"docker0"})
```

Comment:
using util.StringSet instead of a list will clean up some of the checking code (eliminate interfaceSkip, for example)